src/loki/worker.cc

```diff
@@ -44,7 +44,7 @@ void loki_worker_t::parse_locations(google::protobuf::RepeatedPtrField<valhalla:
       else if (location.radius() > max_radius)
         location.set_radius(max_radius);
 
-      if (!location.has_heading_tolerance_case())
+      if (location.has_heading_tolerance_case())
```

Comment:
Not sure I follow this. We're now overriding `heading_tolerance_case` even if it explicitly set?

Reply:
i do not know how that got in here, i didnt even change heading tolarance in the proto. ill add a unit test because obviously this isnt protected by existing tests

Reply:
done